app/views/shared/_two_columns.html.erb

```diff
@@ -119,7 +120,7 @@
               <%= icon "paper-clip", variant: :mini, size: 23, class: 'cursor-pointer text-zinc-600 hover:text-zinc-900 stroke-2 dark:text-gray-300 dark:hover:text-white', title: 'Attach', tooltip: :top %>
             </div>
 
-            <%= form_with(model: [@assistant, @new_message], data: { controller: 'new-message' }) do |form| %>
+            <%= form_with(model: [@assistant, @new_message], data: { controller: "new-message", turbo_frame: "_top" }) do |form| %>
```

Comment:
It's counter-intuitive but without targeting the `_top` frame then morphing won't happen. Instead, the form replaces the `turbo-frame id="conversation"` and it does not morph this replacement.